proc/proc_windows.go

```diff
@@ -61,7 +59,9 @@ func Launch(cmd []string) (*Process, error) {
 		}
 		defer syscall.CloseHandle(syscall.Handle(fd[i]))
 	}
-
+	// create suitable command line for CreateProcess - quote the executable name with absolute path
+	// and pass on the rest of the parameters. see https://msdn.microsoft.com/en-us/library/windows/desktop/ms682425.aspx
+	cmdLine, _ := syscall.UTF16PtrFromString(strings.Join(append([]string{"\"" + argv0Go + "\""}, cmd[1:]...), " "))
```

Comment:
This will not work in many valid scenarios. You should do what standard library does. See https://github.com/golang/go/blob/master/src/syscall/exec_windows.go#L326 for details.